raphtory-graphql/src/main.rs

```diff
@@ -52,29 +53,46 @@ struct Args {
 
     #[arg(long, default_value_t = DEFAULT_AUTH_ENABLED_FOR_READS)]
     auth_enabled_for_reads: bool,
+
+    #[command(subcommand)]
+    command: Option<Commands>,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    #[command(about = "Writes the GraphQL schema into the given path")]
+    Schema {
+        #[arg(value_parser)]
+        path: String,
+    },
 }
```

Comment:
I think starting the server should also be a subcommand so you would do `raphtory_graphql server...` for starting the server and `raphtory_graphql schema ...` for the schema

Reply:
I shall make a ticket for this as discussed in the standup to include this in 0.16.0